### PR TITLE
Contributions Flow Tweaks

### PR DESCRIPTION
**Pull request by Ap0c**
## Why are you doing this?

From both a UX and a legal perspective there are a few small things in the contributions flow that have to be tweaked before we go live.

[**Trello Card**](https://trello.com/c/0YdifOVA/756-contributions-flow-tweaks)

## Changes

- Pulled the legal copy from the bottom of the contributions landing page out into a shared component.
- Applied this shared legal copy to both the one-off and the monthly contributions checkouts.
- Made `the Guardian` use a lowercase 't' on the thank you pages, to fit in correctly with branding.
- Switched to using the word 'Pay' instead of 'Contribute' on the checkout CTAs.
- On the existing contributor page for monthly contributions, updated the link for one-off contributions to point to support rather than contributions frontend. We can now handle one-off contributions in support-frontend (woo!).
- Added the link for 'contact us here' within the legal copy.

## Screenshots

**Checkout Page - Before:**

checkout-before

**Checkout Page - After:**

legal-oneoff

**Thank You Page - Before:**

thankyou-before

**Thank You Page - After:**

thankyou-after

### CODE REVIEW

File: assets/pages/monthly-contributions-existing/monthlyContributionsExisting.jsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ const content = (
           </h2>
           <CtaLink
             text="Make a one-off contribution"
-            url="https://contribute.theguardian.com"
+            url="/oneoff-contributions"
```
Comment (rupertbates) [inline assets/pages/monthly-contributions-existing/monthlyContributionsExisting.jsx]: Yay! 🍰

File: assets/components/stripePopUpButton/stripePopUpButton.jsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ const StripePopUpButton = (props: PropTypes) => {
     <button
       className="component-stripe-pop-up-button"
       onClick={stripeClick}
-    >Contribute with debit/credit card <Svg svgName="credit-card" /></button>
+    >Pay with debit/credit card <Svg svgName="credit-card" /></button>
```
Comment (svillafe) [inline assets/components/stripePopUpButton/stripePopUpButton.jsx]: Probably, in order to make it more generic, we should read the button's text from an argument.

File: assets/components/contribLegal/contribLegal.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+// @flow
```
Comment (svillafe) [inline assets/components/contribLegal/contribLegal.jsx]: We already have another shared legal/copy component (`termsPrivacy`), in order to have a better folder structure, should we create a new folder inside shared component which can group the two pieces of legal copy? What do you think?

Comment (Ap0c) [inline assets/components/contribLegal/contribLegal.jsx]: f48764b

**Pull request merged**